build.xml

```diff
@@ -457,28 +546,50 @@
 
     <phingcall target="check_local_config" />
 
+    <!-- Install / update dependencies -->
+    <phingcall target="composer" />
+    <exec executable="${srcdir}/composer.phar">
```

Comment:
Isn't the composer target already calling composer install? Do we need to explicitly call it again here?

I also wonder if we should be downloading composer.phar into ${srcdir}/downloads so it's not cluttering up the root of the project... though that's probably just scope creep at this point and is better addressed in a follow-up. :-)

Reply:
Of course, my mistake.

I decided to move composer.phar to downloads now while at it.

Reply:
Yeah, that was a weirder mess than I had realized; worth sorting out!